packages/@n8n/db/src/migrations/common/1760116750277-CreateOAuthEntities.ts

```diff
@@ -0,0 +1,109 @@
+import type { MigrationContext, ReversibleMigration } from '../migration-types';
+
+export class CreateOAuthEntities1760116750277 implements ReversibleMigration {
+	async up({ schemaBuilder: { createTable, column } }: MigrationContext) {
+		// Create oauth_clients table
+		await createTable('oauth_clients').withColumns(
+			column('id').varchar(255).primary.notNull,
+			column('name').varchar(255).notNull,
+			column('redirectUris').json.notNull,
+			column('grantTypes').json.notNull,
+			column('clientSecret').varchar(255),
+			column('clientSecretExpiresAt').int,
```

Comment:
Can we add a comment about what this integer represents. Is it a unix timestamp?